releases/pre-release/docker-compose-pre-release-no-secty-arm64.yml

```diff
@@ -63,17 +64,19 @@ services:
   command:
     container_name: edgex-core-command
     depends_on:
-    - consul
-    - database
-    - metadata
+      consul:
+        condition: service_started
+      database:
+        condition: service_started
+      metadata:
+        condition: service_started
```

Comment:
This is an issue with using the latest docker-compose to generate the compose files. Please revert your version of docker-compose to version `1.27.2`

Reply:
Regenerated the compose file.